polaris/dataset/_factory.py

```diff
@@ -221,20 +221,21 @@ def add_columns(
             adapter = adapters.get(name)
             self.add_column(series, annotation, adapter)
 
-    def add_from_file(self, path: str):
+    def add_from_file(self, path: str, append: bool = False):
```

Comment:
**Suggestion**: Do we need a _new_ parameter? Could we use `axis` here as well ?

Reply:
`append` was already added in 'convert' function. This parameter should be removed from here. 
My mistake.